src/app/screens/Publishers.tsx

```diff
@@ -20,9 +20,14 @@ function Publishers() {
       const response = await utils.call<{
         allowances: Allowance[];
       }>("listAllowances");
+      const response2 = await utils.call<{
+        blocklist: Blocklist[];
+      }>("listBlocklist");
       const allowances = response.allowances.map((allowance) => {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        let retobj: any = allowance;
```

Comment:
Would be great to avoid `any` if possible.

Reply:
please check my attempt at fixing this, I added a Publisher type.